cmd/resourcegen/embed/resources/$cloud_provider$/$filename$.go.tmpl

```diff
@@ -0,0 +1,146 @@
+package {{ .CloudProvider }}
+
+import (
+	"fmt"
+
+	"github.com/shopspring/decimal"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// TODO: fill in these {{ .ResourceName }} comments with a short description of the cloud resource.
+// Add any links to the pricing pages or documentation that might be useful to developers in the future.
+// e.g:
+// 		more resource information here: https://aws.amazon.com/<PATH/TO/RESOURCE>/
+//
+//		pricing information here: https://aws.amazon.com/<PATH/TO/PRICING>/
+//
+type {{ .ResourceName }} struct {
+	{{- if .WithHelp }}
+	// Add {{ .ResourceName }} struct properties below that correspond to your resource
+	// and are directly used in cost calculation.
+	//
+	// These properties are configured directly from attributes parsed in the IAC provider.
+	// See your resource file: internal/providers/terraform/{{ .CloudProvider }}/{{ .Filename }}.go
+	// for an example in how this is achieved.
+	//
+	// Below are a few examples of common properties that are often part of resources:
+	{{- end }}
+	// Address is the unique name of the resource in the IAC language.
+	Address       string
+	// Region is the {{ .CloudProvider }} region the {{ .ResourceName }} is provisioned within.
+	Region        string
+	{{- if .WithHelp }}
+	// InstanceCount is an example properties designed to show you a complete example with cost components.
```

Comment:
Not sure if this comment should list all three properties, or just InstanceCount.
```suggestion
	// InstanceCount is an example property designed to show you a complete example with cost components.
```